internal/clients/nightfall/nightfall.go

```diff
@@ -62,9 +65,11 @@ type contentToScan struct {
 }
 
 func foundSensitiveData(finding nightfallAPI.ScanResponse, detectorConfigs nightfallconfig.DetectorConfig) bool {
-	minimumLikelihoodForDetector := detectorConfigs[nightfallAPI.Detector(finding.Detector)]
```

Comment:
Prior implementation was also returning true if the finding detector wasn't in detectorConfigs. 
`likelihoodThresholdMap[minimumLikelihoodForDetector]` would default to zero, causing the return value to always evaluate to true.